### ISSUE TITLE
Respect Pimcore fallback languages

### ISSUE THREAD

**Issue by valentin-** [dachcom-digital/pimcore-seo#44]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | no
| BC Break report? | no
| RFC?             | no

Hi, I'm currently working on a project with different regions/languages and fallback languages, f.e:

- en
- de-at -> fallback to de
- de-de -> fallback to de

The fallback languages are useful if the same news object is used in different regions (Austria, Germany). Therefore only the title for de will be set (in the data object), and the value for de-at/de-de will be empty.

It would be handy if this feature also applies to the seo settings. Currently the value for de_AT/de_DE must be set, de won't get used as fallback.

![image](https://user-images.githubusercontent.com/90033/223129147-abb6d00f-6d98-4c99-989d-0e6bd00d8b1a.png)

What do you think about this feature?

Thanks in advance.

**Comment by solverat** [comment]
You're right, this should be possible!

**Comment by solverat** [comment]
@valentin- I just created a PR for that. 

But I'm not sure if we should implement a flag to enable this feature, since this could be treated as a breaking change (new output).

Something like this maybe?

```yaml
seo:
  meta_data_integrator:
    allow_locale_fallback: true # default false
```

**Comment by solverat** [comment]
We've just discussed this internally and have come to the conclusion that it does not need an additional configuration flag. This should be the standard behavior.

**Comment by solverat** [comment]
Fixed with #47
